src/sunmatrix/sparse/sunmatrix_sparse.c

```diff
@@ -618,18 +622,19 @@ int SUNMatScaleAddI_Sparse(sunrealtype c, SUNMatrix A)
     N = SM_ROWS_S(A);
   }
 
-  /* access data arrays from A (return if failure) */
+  /* access data arrays from A */
   Ap = Ai = NULL;
   Ax      = NULL;
-  if (SM_INDEXPTRS_S(A)) { Ap = SM_INDEXPTRS_S(A); }
-  else { return (SUNMAT_MEM_FAIL); }
-  if (SM_INDEXVALS_S(A)) { Ai = SM_INDEXVALS_S(A); }
-  else { return (SUNMAT_MEM_FAIL); }
-  if (SM_DATA_S(A)) { Ax = SM_DATA_S(A); }
-  else { return (SUNMAT_MEM_FAIL); }
-
-  /* determine if A: contains values on the diagonal (so I can just be added in);
-     if not, then increment counter for extra storage that should be required. */
+  Ap      = SM_INDEXPTRS_S(A);
```

Comment:
```suggestion
  Ap = SM_INDEXPTRS_S(A);
```

Reply:
The formatter did this because its aligning the second `=` with the `=` on the previous line. The way to "fix" it is:
```c
// Change this
Ap = Ai = NULL;
Ax      = NULL;
Ap      = SM_INDEXPTRS_S(A);

// To this
Ap = NULL;
Ai = NULL;
Ax = NULL;
Ap = SM_INDEXPTRS_S(A);
```